docs/storage/fleek-sdk.md

```diff
@@ -0,0 +1,210 @@
+---
+date: "1"
+
+---
+The Fleek SDK is a tool that allows you to interact programmatically with two features on the Fleek platform **IPFS Pinning** and **Internet Computer (IC) Storage**. It is an alternative to the Fleek Storage JS (the Storage JS uses S3/MinioDB to interact with Fleek's Storage/IPFS products, while the Fleek SDK doesn't).
```

Comment:
I'd say IPFS APIs instead of IPFS Pinning.

Reply:
> It is an alternative to the Fleek Storage JS (the Storage JS uses S3/MinioDB to interact with Fleek's Storage/IPFS products, while the Fleek SDK doesn't).

It's more decentralised alternative to our S3 Storage which is using Minio and centralised db for storing metadata.